extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -531,6 +541,80 @@ export class AmpIframe extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Requests consent data from consent module
+   * and forwards information to iframe
+   * @param {Window} source
+   * @param {string} origin
+   * @private
+   */
+  sendConsentData_(source, origin) {
+    const consentPolicyId = super.getConsentPolicy() || 'default';
+    const consentStringPromise = this.getConsentString_(consentPolicyId);
+    const metadataPromise = this.getConsentMetadata_(consentPolicyId);
+    const consentPolicyStatePromise = this.getConsentPolicyState_(
```

Comment:
nit: You can get rid of these intermediate methods, just call `getConsentPolicyInfo`, etc. directly.